packages/core/upload/admin/src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -19,13 +20,21 @@ export const Breadcrumbs = ({ breadcrumbs, ...props }) => (
               .splice(index + 1, breadcrumbs.length - 1)
               .map(parent => parent.id)}
             key={`breadcrumb-${crumb?.id ?? 'menu'}`}
+            currentFolderId={currentFolderId}
+            onChangeFolder={onChangeFolder}
           />
         );
       }
 
-      if (crumb.href) {
+      if (crumb.href !== undefined) {
         return (
-          <CrumbLink key={`breadcrumb-${crumb?.id ?? 'root'}`} as={NavLink} to={crumb.href}>
+          <CrumbLink
+            key={`breadcrumb-${crumb?.id ?? 'root'}`}
+            as={onChangeFolder ? 'button' : NavLink}
+            to={onChangeFolder ? undefined : crumb.href}
+            onClick={onChangeFolder && (() => onChangeFolder(crumb.id))}
+            aria-label={crumb?.label}
```

Comment:
Why do we need to repeat the label?

Reply:
I thought it would be clearer in case it is a button (for the CM) but maybe not needed indeed 🤔